control/cli.py

```diff
@@ -166,13 +166,15 @@ def delete_bdev(self, args):
     @cli.cmd([
         argument("-n", "--subnqn", help="Subsystem NQN", required=True),
         argument("-s", "--serial", help="Serial number", required=True),
+        argument("-m", "--max_namespaces", help="Maximum Namespaces", type=int, default=0, required=False),
```

Comment:
I'd avoid mixing dashes and underscores and follow `--gateway-name` example:
```suggestion
        argument("-m", "--max-namespaces", help="Maximum number of namespaces", type=int, default=0, required=False),
```